### PR TITLE
main/libuv: upgrade to 1.30.1

### PR DESCRIPTION
**Pull request by OTLabs**
- https://github.com/libuv/libuv/releases 1.30.1

ABI compatible https://abi-laboratory.pro/?view=timeline&l=libuv

### CODE REVIEW

**Comment by OTLabs** [comment]
@ncopa Could you please check proposed changes? Thank you in advance!

**Comment by andypost** [comment]
https://github.com/libuv/libuv/releases 1.30.1 released and required for https://github.com/alpinelinux/aports/pull/9301

File: main/libuv/APKBUILD [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libuv
-pkgver=1.29.1
+pkgver=1.30.0
```
Comment (andypost) [inline main/libuv/APKBUILD]: should be 1.30.1 for nodejs-current

Comment (andypost) [reply main/libuv/APKBUILD]: just squash commits, please

Comment (OTLabs) [reply main/libuv/APKBUILD]: done!

**Pull request closed without merging**